static/app/components/core/textarea/index.tsx

```diff
@@ -27,24 +29,22 @@ export interface TextAreaProps
 }
 
 const TextAreaControl = forwardRef(function TextAreaControl(
-  {autosize, rows, maxRows, size: _size, ...p}: TextAreaProps,
+  {autosize, rows = 3, maxRows, size: _size, ...p}: TextAreaProps,
```

Comment:
as discussed with @Jesse-Box, textareas will now have 3 rows per default to make them look visually different from normal inputs.

this has no effect in the old theming because there, we set the height manually. In chonk, I’ve removed the explicit height setting to let `rows` control the height.